charts/nr-k8s-otel-collector/README.md

```diff
@@ -1,75 +1,107 @@
-# Installation: 
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
 
-### 1. Download source code 
-```
-git clone https://github.com/newrelic/helm-charts.git
-```
+# nr-k8s-otel-collector
+
+A Helm chart to monitor a Kubernetes Cluster using an OpenTelemetry Collector.
+
+# Helm installation
 
-### 2. Update config [here](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) to add a cluster name, and New Relic Ingest - License key
-Example: 
+Download and Update config [here](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) to add a cluster name, and New Relic Ingest - License key
+
+Example:
 ```
 licenseKey: "EXAMPLEINGESTLICENSEKEY345878592NRALL"
-cluster: "SampleApp" 
+cluster: "SampleApp"
 ```
 
-### 3. From the root directory of this chart, run:
-```
-helm install nr-k8s-otel-collector nr-k8s-otel-collector -n newrelic --create-namespace
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://helm-charts.newrelic.com
+helm upgrade --install newrelic/nr-k8s-otel-collector -f your-custom-values.yaml  -n newrelic --create-namespace
 ```
 
 ## Confirm installation
-### Watch pods spin up: 
+### Watch pods spin up:
+
 ```
-kubectl get pods -A --watch 
+kubectl get pods -n newrelic --watch
```

Comment:
Does the pods always come up in the newrelic namespace?

Reply:
Yes it will if they follow the installation command from above.